ssh/options.c

```diff
@@ -1,14 +1,16 @@
-/* Generated by Cython 0.29.21 */
+/* Generated by Cython 0.29.26 */
```

Comment:
Going to need to re-generate all the *.c files when changing Cython versions. `rm -f ssh/*.c; python setup.py build_ext -i`

Reply:
I've regenerated it with the committed version for this PR to keep changes local and make another one for bumping the Cython version